distribute/templates/distribute_job.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: distributor
 
 spec:
-  schedule: "*/1 * * * *"
+  schedule: "* */1 * * *"
```

Comment:
Fint. Det dumme med 1mins intervaller var at kubectl get events ble oversvømt av greier, så det var vanskelig å finne andre errors. Men det er litt kubectl get event sin ikke-tidssorterte liste sin skyld. Men det går an å fikse med `kubectl get events --sort-by='.lastTimestamp'` så jeg nå.

Reply:
Yeah. I første omgang, så tenkte jeg rett og slett at det ikke var behov for å publisere oppdateringer hvert minutt.